Artifacts/PgSql/Structure/Admin/0161-Rename-OdsInstanceDerivative.sql

```diff
@@ -0,0 +1,50 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+ALTER TABLE dbo.OdsInstanceDerivative
+	DROP CONSTRAINT IF EXISTS FK_OdsInstanceDerivative_OdsInstanceId_OdsInstanceId;
+
+ALTER TABLE dbo.OdsInstanceDerivative
+	DROP CONSTRAINT IF EXISTS UC_OdsInstanceDerivative_OdsInstanceId_DerivativeType;
+
+ALTER TABLE dbo.OdsInstanceDerivative
```

Comment:
IF EXISTS (SELECT column_name
               FROM information_schema.columns
               WHERE table_name = 'OdsInstanceDerivative'
                 AND column_name = 'OdsInstanceId') THEN
        ALTER TABLE OdsInstanceDerivative
        RENAME COLUMN OdsInstanceId TO OdsInstance_OdsInstanceId;
    END IF;